src/server/ua_gds_pushmanagement_ns0.c

```diff
@@ -0,0 +1,161 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright (c) 2023 Fraunhofer IOSB (Author: Noel Graf)
+ */
+
+#include "ua_gds_pushmanagement_ns0.h"
```

Comment:
Move the information model to a second PR when you have some usable functionality.

Also this should be behind another build flag.
Not just UA_ENABLE_ENCRYPTION.